metadata-db-app/src/cmr/metadata_db/data/memory_db.clj

```diff
@@ -302,7 +302,7 @@
   involve the passed in concept id."
   [concepts concept-id]
   (util/remove-nils-empty-maps-seqs
-   (merge
```

Comment:
There are two issues with using merge: 
1. (merge [] [1 2 3]) returns [[1 2 3]], rather than [1 2 3].
2. (merge [1 2 3] [4 5 6]) returns [1 2 3 [4 5 6]], rather than [1 2 3 4 5 6].
We need both situations to work correctly.

Reply:
1. (reduce conj [][1 2 3])
2.(reduce conj [1 2 3] [4 5 6])

Reply:
concat works fine too. Thanks.